File: Software Technologies/Practical Exam/csharp/CatShop/Models/Cat.cs
namespace CatShop.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Cat
    {
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Nickname { get; set; }

        public double Price { get; set; }
    }
}

File: Software Technologies/Practical Exam/js/models/cat.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Cat = sequelize.define("Cat",{
        name:{
            type: Sequelize.TEXT,
            allowNull: false,
            required:true
        },
        nickname:{
            type: Sequelize.TEXT,
            allowNull: false,
            required:true
        },
        price:{
            type: Sequelize.DOUBLE,
        }

    },{
        timestamps:false
    });

    return Cat;
};